### ISSUE TITLE
Unicode smilies/icons cause red explamation mark (ie top-most feature to fail)

### ISSUE THREAD

**Issue by kostasvl** [tabfloater/tabfloater#264]
## Description
Hi. I noticed that unicode smilies/icons cause the red explamation mark to appear in this extension's icon, which appears to be because the top-most flag fails to be applied to the window.

![image](https://user-images.githubusercontent.com/17363218/146686239-67809a9f-e27b-47b2-b5f9-7c7bc6208082.png)

![image](https://user-images.githubusercontent.com/17363218/146686244-3cdbf13a-c463-4148-a85d-0873241327a9.png)

I found this in the log:

> 2021-12-19 18:19:44.328 (   0.004s) [main thread     ]tabfloater_companion.cp:60    INFO| TabFloater Companion started. Version: 1.2.0, OS: Windows, Git commit: 1ab502c80b6e632f562a5aa6499f70f769179fd1
2021-12-19 18:19:44.328 (   0.004s) [main thread     ]tabfloater_companion.cp:221   INFO| Input JSON: "{"action":"setAlwaysOnTop","windowTitle":"I’m not feeling Christmassy yet! 🎄 - YouTube","parentWindowTitle":"New Issue · tabfloater/tabfloater","debug":"true"}"
2021-12-19 18:19:44.328 (   0.004s) [main thread     ]tabfloater_companion.cp:222   INFO| Action: "setAlwaysOnTop"
2021-12-19 18:19:44.328 (   0.005s) [main thread     ]tabfloater_companion.cp:231   INFO| Window title: "I’m not feeling Christmassy yet! 🎄 - YouTube"
**2021-12-19 18:19:44.329 (   0.005s) [main thread     ]tabfloater_companion.cp:250    ERR| An error occurred while manipulating window: wstring_convert::to_bytes**
2021-12-19 18:19:44.329 (   0.005s) [main thread     ]tabfloater_companion.cp:148   INFO| Sending JSON: "{"status":"error"}", length: 18

## Reproduction steps
I can repro this 100% with either of these two urls:

- https://www.youtube.com/watch?v=UPICfwDirEY
- https://twitter.com/purrrmeister/status/1472561883152629764

I'm using "Remember position and size", if that makes a difference. Big fan of this feature, thanks ;)

## Environment
[NOTE]: # ( Provide information about your machine and environment. )
[NOTE]: # ( For "Companion installation", specify how you installed the Companion. E.g. for Windows: installer/portable, for Linux: AppImage/PPA )
 * **TabFloater version:** 1.2.0, I think?
 * **Companion version:**  1.2.0 (Windows)
 * **Operating system:** Windows 10
 * **Browser**: Chrome
 * **Companion installation:** ..?

**Comment by ba32107** [comment]
Hi! Sorry for the late reply. I don't have much time during the holiday season. I'll try to have a look at this problem as soon as possible.

**Comment by kostasvl** [comment]
Not a problem at all. Thank you very much! :)

**Comment by ba32107** [comment]
I tried to reproduce this on my Linux machine, and I wasn't able to. Both these sites work for me. The problem comes from the companion application though, and that is operating system dependent, and I noticed you're using Windows. I also have a Windows machine, so I'll try it there as well later this week to see if I can reproduce it.

> Companion installation: ..?

Btw, this is supposed to mean "how is it installed", for example via installer, portable, etc. But it shouldn't make a difference for this issue.
